contracts/staking/cw20-stake-external-rewards/src/state.rs

```diff
@@ -6,8 +6,6 @@ use cw_storage_plus::{Item, Map};
 
 #[cw_serde]
 pub struct Config {
-    pub owner: Option<Addr>,
```

Comment:
i'd consider also renaming the config key to "config_v2"? for other migrations we've done this to avoid overriding state just in case

Reply:
great point. thanks. updated